draft-ietf-keytrans-protocol.md

```diff
@@ -1805,6 +1772,91 @@ each label are not provided, given that these can be cached from the
 original Search or Update query.
 
 
+# Third Parties
+
+## Management
+
+With the Third-Party Management deployment mode, a third party is responsible
+for the majority of the work of storing and operating the Transparency Log,
+while the Service Operator serves mainly to enforce access control and
+authenticate the addition of new entries. All user queries specified in
+{{user-operations}} are initially sent by users directly to the Service Operator
+to be forwarded to the Third-Party Manager if they pass access control.
+
+While other operations are forwarded by the Service Operator unchanged,
+`UpdateRequest` structures are forwarded to the Third-Party Manager with the
+Service Operator's signature attached:
+
+~~~ tls-presentation
+struct {
+  UpdateRequest request;
+  opaque signature<0..2^16-1>;
+} ManagerUpdateRequest;
+~~~
+
+The signature is computed as described in {{update-format}}. The Service
+Operator MUST maintain its own records (independent of the Third-Party Manager)
+for the greatest version of each label for the purpose of producing this
+signature.
+
+## Auditing
```

Comment:
I think this section deserves a bit more context as to how auditing is supposed to work out, e.g., say that "auditors will check append-only property at a regular basis," etc.

Reply:
Background info I think is more appropriate for the arch document. I looked in the arch docu and we're pretty thin on how we describe auditing there too, so I will flesh it out separately

Reply:
We have a lot of introductory/glue text in other sections already, and I think this is a strength of the draft. Why not just a small intro to third-party management and auditing as intro text to the section "Third Parties"? I think we could be very succinct here, but this would help readers.